include/vcpkg/archives.h

```diff
@@ -13,16 +13,36 @@
 
 namespace vcpkg
 {
+    enum class ExtractionType
+    {
+        UNKNOWN,
```

Comment:
Normally enum class members are CamelCase. Though I'm not sure this enum should exist...